requirements.txt

```diff
@@ -7,7 +7,7 @@ matplotlib~=3.0
 networkx~=2.4
 numpy~=1.16, < 1.19
 pandas
-protobuf~=3.11.0
+protobuf~=3.12.0
```

Comment:
Do we want this to be 3.12 to possibly pick up bug fix releases?

Reply:
In my understanding the `~=` does exactly that: https://www.python.org/dev/peps/pep-0440/#compatible-release
I.e. `~=3.12.0` is equivalent to `>= 3.12.0, == 3.12.*`. We could argue that maybe can start at 3.12.3 as that's today's latest, but I thought we can live with that lower boundary to reduce noise for those who have [3.12.0. 3.12.2] installed.